libs/frontend/util-command/src/lib/command.ts

```diff
@@ -17,6 +40,30 @@ const pushCommandHelper = (site: ISite, command: ICommand, clearRedo = true) =>
   }
 }
 
+const commandsInGroup = (command: ICommand, commandTypes: CommandType[]): boolean => {
+  if (command.type === CommandType.Group) {
+    const groupCommands = command.data as ICommandGroupData
+    return groupCommands.commands.some((cmd) => commandTypes.includes(cmd.type))
+  }
+  return false
+}
+
+const shouldCloseMixinMenuForUser = (newCmd: ICommand | undefined): boolean => {
+  const mixinCommands = [CommandType.SetMixinEntry, CommandType.EditStyleMixin]
+  if (
+    !newCmd ||
+    mixinCommands.includes(newCmd.type) ||
+    commandsInGroup(newCmd, mixinCommands)
```

Comment:
@abemscac I added these two lines, otherwise changing the name or style value in the mixin menu caused it to close! I hope I understood everything correctly.

Reply:
Oh, I didn't notice this, sorry about that!
Thanks for the fix!